.rubocop/custom.yml

```diff
@@ -108,7 +108,7 @@ Rails/I18nLazyLookup: # new in 2.14
 Rails/I18nLocaleAssignment: # new in 2.11
   Enabled: true
 Rails/I18nLocaleTexts: # new in 2.14
-  Enabled: true
+  Enabled: false
```

Comment:
we only will have one locale for deployment and there's not a need to support more languages;  I was getting rubocop errors.